modules/apps/frontend-theme/kyoung-liferay-theme/src/js/canvas.js

```diff
@@ -0,0 +1,89 @@
+var space;
+
+function floatySpace() {
+    var colors = [
+        '#FF3F8E', '#04C2C9', '#2E55C1'
+    ];
+
+    space = new CanvasSpace('canvas', '#252934').display();
+    var form = new Form(space);
+
+	// Elements

```

Comment:
JS comments should be indented to the same level as to which it is describing.  JS comments are not necessary unless they are descriptive and useful.